columnflow/production/cms/btag.py

```diff
@@ -135,6 +140,35 @@ def add_weight(syst_name, syst_direction, column_name):
     return events
 
 
+def get_b_score_column(btag_config: tuple[str, list, str]) -> str:
```

Comment:
I think if you support the old structure `tuple[str, list]` you should also form the type hints accordingly, i.e.

```suggestion
def get_b_score_column(btag_config: tuple[str, list, str] | tuple[str, list]) -> str:
```